build.gradle

```diff
@@ -256,3 +256,27 @@ task updateRootDocs << {
 }
 
 build.dependsOn('updateRootDocs')
+
+apply plugin: 'maven-publish'
+
+version = System.getenv("PACKAGE_VERSION") ?: "0.0.1-snapshot"
```

Comment:
`version = 1.0.0.BUILD-SNAPSHOT` is defined in `gradle.properties` -- can we use that instead? That's what `install` in `publish-maven.gradle` uses.